### PR TITLE
(REF) Tidy up unused params in api_v3_ContributionSoftTest

### PR DESCRIPTION
**Pull request by braders**
Overview
----------------------------------------
Bit of test cleanup here, which also helps on the cleanup of dyanmic properties for PHP8.2

Before
----------------------------------------
1. `_processorParams` set but not used (and I also checked the parent class, plus for signs of it being passed around in odd ways). Looks like a copy-paste from some other tests.
2. `_softcontribution` and `_softcontribution2` were properties, but could have just been straight variables. This is test code, so no backward-compataiblility risk to changing these.
3. Some variables were being set but never used.

After
----------------------------------------
The above issues resolved. This class is now PHP8.2 ready.

### CODE REVIEW

**Comment by seamuslee001** [comment]
@braders 

Test failure relates

```
api_v3_ContributionSoftTest::testGetContributionSoft
Trying to access array offset on value of type int

/home/jenkins/bknix-dfl/build/core-25322-5lef8/web/sites/all/modules/civicrm/tests/phpunit/api/v3/ContributionSoftTest.php:108
/home/jenkins/bknix-dfl/build/core-25322-5lef8/web/sites/all/modules/civicrm/tests/phpunit/CiviTest/CiviUnitTestCase.php:235
/home/jenkins/bknix-dfl/extern/phpunit8/phpunit8.phar:1721
```

**Comment by eileenmcnaughton** [comment]
Looks like a good improvement (once tests are passing)

Non blocking but it would be nice to rename `softcontribution` to `softContribution` - the more we comply with IDE spelling checkers the easier to spot real misspellings & also non-native English speakers can find it hard to parse where one word ends & another starts even when it seems easy to us

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by braders** [comment]
test this please

**Comment by seamuslee001** [comment]
test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by yashodha** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
that last one seems to no longer run so I think all is passing
